test/client_test.cc

```diff
@@ -37,7 +37,7 @@ TEST_F(ClientTest, AutoConcurrencyRun) {
   argv.push_back("--duration");
   argv.push_back("1");
   argv.push_back("--rps");
-  argv.push_back("1");
+  argv.push_back("10");
```

Comment:
For my understanding, why was this change necessary?

Reply:
It wasn't needed, accidentally left it in. Backed it out.